core/shared/src/main/scala/fs2/Pipe.scala

```diff
@@ -22,7 +22,7 @@ object Pipe {
 
     // Steps `s` without overhead of resource tracking
     def stepf(s: Stream[Read,O]): Read[UO] = {
-      Algebra.runFold(
+      Algebra.compile(
```

Comment:
Why isn't this `compile.fold` (the same for Pipe2)?

Reply:
This was a hold over from an earlier version -- with the current code, this can be reimplemented in terms of `pull.uncons` and `streamNoScope`. Fixed.